spec/helpers/i18n_helper_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+RSpec.describe I18nHelper do
+  describe '#date_eras_gregorian' do
+    context 'before christ' do
+      it 'converts gregorian before christ to correct format' do
+        ['-1234', ' -1234', '-1234 ', ' -1234 ', '- 1234', ' - 1234', '- 1234 ', ' - 1234 '].each do |date|
+          formatted_date = helper.date_eras_gregorian(date)
+          expect(formatted_date).to eq('1234 BCE'), "date = '#{date}' has failed"
```

Comment:
The default RSpec failure message, e.g. 
```
       expected: "66 CE"
            got: "066"
```
is more helpful than this custom one, so I suggest removing the latter.